Dockerfile

```diff
@@ -21,16 +21,17 @@ RUN mkdir -p /runner/inventory /runner/project /runner/artifacts /runner/.ansibl
 	chmod -R g+w /runner && chgrp -R root /runner && \
 	chmod g+w /etc/passwd
 
-VOLUME /runner/inventory
-VOLUME /runner/project
-VOLUME /runner/artifacts
+VOLUME /runner
 
 ENV LANG=en_US.UTF-8
 ENV LANGUAGE=en_US:en
 ENV LC_ALL=en_US.UTF-8
 ENV RUNNER_BASE_COMMAND=ansible-playbook
 ENV HOME=/runner
 
+ENV ANSIBLE_STDOUT_CALLBACK=awx_display
+ENV ANSIBLE_CALLBACK_PLUGINS=/usr/lib/python3.6/site-packages/ansible_runner/callbacks
```

Comment:
@AlanCoding Didnt you have some thoughts about putting the callback plugin into a collection? If we did that, then we wouldn't need this, right?

Reply:
yes, I would absolutely like to do that

I went over what I wanted to do with Shane, and didn't see any of this container-isolation stuff as something which would fundamentally change a collection conversion.

This line, however, is something that I don't think we need. Down below, I see that `ansible-runner` gets called. If that's the entry point, then we shouldn't be setting anything like this beforehand.

That gets set in the runner code itself, only for the `ansible-playbook` process

https://github.com/ansible/ansible-runner/blob/f73705bf9b09424c9e5353d7f5119147e844eaa5/ansible_runner/runner_config.py#L198

you don't need to have it set in the process where `ansible-runner` is invoked. So just delete these 2 lines.

Reply:
If we do that then lets discuss it outside of the context of this PR since it's a pretty major component for Runner itself.

Reply:
Clarification: we should remove these settings, because they are set in the code itself.